astropy/modeling/tests/test_input.py

```diff
@@ -231,22 +231,21 @@ def test_nonlinear_lsqt_Nset_2d(self, fitter):
         """1 set 2d x, 1set 2D y, 2 param_sets, NonLinearFitter"""
         fitter = fitter()
 
-        MESSAGE = (
-            r"Input argument .* does not have the correct dimensions in .* for a model"
-            r" set with .*"
+        MESSAGE = r"Non-linear fitters can only fit one data set at a time"
```

Comment:
Why would the error message change?

Reply:
@WilliamJamieson requested the message change. https://github.com/astropy/astropy/pull/16158#discussion_r1513482380